File: Algorithm/Backjun/bronze5/b1330/b1330.py
# 두 수 비교하기
# https://www.acmicpc.net/problem/1330

a, b = map(int, input().split())

# 두 정수를 비교하고 결과 출력
def compare(a, b):
    if a > b:
        return('>')
    elif a < b:
        return('<')
    else:
        return('==')

print(compare(a, b))

File: Algorithm/Backjun/bronze5/b1330/b1330.js
// 두 수 비교하기
// https://www.acmicpc.net/problem/1330

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// 두 정수를 비교하고 결과 출력
function compare(a, b) {
  if (a > b) {
    return ">";
  } else if (a < b) {
    return "<";
  } else {
    return "==";
  }
}

rl.question('', (input) => {
  const [a, b] = input.split(' ').map(Number);

  console.log(compare(a, b));

  rl.close();
});
